src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj

```diff
@@ -17,8 +17,11 @@
     <PackageReference Include="Azure.Data.AppConfiguration" Version="1.3.0-beta.2" />
     <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.3.0" />
+    <PackageReference Include="DnsClient" Version="1.7.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.18" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.18" />
+    <PackageReference Include="System.Linq.Async" Version="6.0.1" />
```

Comment:
what is system.linq.async for ?

Reply:
Two purposes:
1. For evaluating empty of async enumerable avaliableClients, if it's empty, would log and return.
2. For using `MoveNextAsync()` with CancellationToken in signature 

But by looking back, seems we don't really need this package.
For #1, we can use an enumerator to check it by `MoveNextAsync()`
For #2, we don't really need to pass cancellationToken while iterating it, we manually throw with `cancellationToken.ThrowIfCancellationRequested()`  in `ExecuteWithFailOverPolicyAsync()`

Reply:
If we can avoid the package reference great !